Form/Type/PhoneNumberType.php

```diff
@@ -81,7 +81,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 $choiceType = 'Symfony\\Component\\Form\\Extension\\Core\\Type\\ChoiceType';
                 $textType = 'Symfony\\Component\\Form\\Extension\\Core\\Type\\TextType';
                 $countryOptions['choice_translation_domain'] = false;
-                $countryOptions['choices_as_values'] = true;
+
+                // To be removed when dependency on Symfony Form is bumped to 3.1.
+                if (\Symfony\Component\HttpKernel\Kernel::VERSION_ID < 30100) {
```

Comment:
Using the version from the HttpKernel is not a good idea as the installed versions of the Form and the HttpKernel components may differ.